python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -374,6 +374,11 @@ def __init__(self, settings: Mapping[str, Any], pre_sensor_interval_seconds: int
     def daemon_type(cls) -> str:
         return "ASSET"
 
+    def instrument_elapsed(
+        self, sensor: Optional[RemoteSensor], elapsed: Optional[float], min_interval: Optional[int]
```

Comment:
i don't think min_interval needs to be optional anymore? It's always non-None if there's a sensor and _pre_sensor_interval_seconds is never None